src/ethereum_test_forks/base_fork.py

```diff
@@ -258,6 +282,16 @@ def solc_name(cls, block_number: int = 0, timestamp: int = 0) -> str:
         """
         pass
 
+    @classmethod
+    def is_supported_by_solc(cls, version: Version) -> bool:
+        """
+        Returns True if the fork is supported by solc (--evm-version).
+        """
+        assert (
+            version in SOLC_VERSION_TO_SUPPORTED_FORKS.keys()
```

Comment:
It's really interesting that this works even if you parse a very specific build, e.g., `0.8.20+commit.a1b79de6.Linux.g++`, it will match 0.8.20. `semver` librrary magic.